include/picongpu/fields/incidentField/profiles/ExpRampWithPrepulse.hpp

```diff
@@ -188,10 +188,6 @@ namespace picongpu
                     private:
                         HDINLINE float_X getEnvelope(float_X const runTime) const
                         {
-                            /* workaround for clang 5 linker issues
-                             * `undefined reference to
-                             * `picongpu::fields::laserProfiles::ExpRampWithPrepulseParam::INT_RATIO_POINT_1'`
-                             */

```

Comment:
only the comment was removed but not the actual workaround - @psychocoderHPC could you comment on that?

Reply:
The implementation is OK, we are required in someplace to store the constexpr in an intermediate variable before. I assume that clan5 in the past had a name lookup issue and used always the wrong variable.